frontend/src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App() {
       <Button type={ButtonOptions.Plus}></Button>
       <Button type={ButtonOptions.Bookmark}></Button>
       <Button type={ButtonOptions.String}>Log in</Button>
+      <CreateEvent></CreateEvent>
```

Comment:
in the not so distant future we will probably change the layout of this component a bit so that its a generic 'settings page' which has the heading and stuff you've already designed, but then <CreateEvent> will be one of its children. and that will just be a form. but for now this is okay 👍️👍️👍️👍️👍️